package.json

```diff
@@ -26,15 +26,22 @@
     "lang:refresh": "node scripts/refresh-language.js",
     "csl:refresh": "node scripts/update-csl.js",
     "reveal:build": "node scripts/update-reveal.js",
-    "wp:dev": "cross-env NODE_ENV=development webpack --progress --colors",
-    "wp:prod": "cross-env NODE_ENV=production webpack --progress --colors",
-    "watch:gulp": "gulp watch",
```

Comment:
I think the gulpfile (as well as a few related dev dependencies) can be removed as well, right?

Is electron-webpack taking care of everything done in these scripts? E.g. linting the files etc

Reply:
Yes, you are right. I had this it removed locally, I just haven't updated it. I pushed it now.